ci/travis/install-dependencies.sh

```diff
@@ -24,7 +24,7 @@ if [[ "$PYTHON" == "2.7" ]] && [[ "$platform" == "linux" ]]; then
   wget https://repo.continuum.io/miniconda/Miniconda2-4.5.4-Linux-x86_64.sh -O miniconda.sh -nv
   bash miniconda.sh -b -p $HOME/miniconda
   export PATH="$HOME/miniconda/bin:$PATH"
-  pip install -q cython==0.29.0 cmake tensorflow gym==0.10.11 opencv-python pyyaml pandas==0.23.4 requests \
```

Comment:
Is this change intentional? I think we want to pin `gym` for Python 2.7 until the next release. See the discussion in https://github.com/ray-project/ray/pull/3973.

Reply:
oops I did not notice that change. I should have undid it